.readthedocs.yml

```diff
@@ -1,5 +1,17 @@
+version: 2
+
+sphinx:
+  configuration: docs/source/conf.py
+
+build:
+  os: ubuntu-20.04
+  tools:
+    python: "3.10"
```

Comment:
This was previously set to python 3.5 (i.e. oldest version supported) and this sets it to 3.10 (i.e. newest version supported). 

Any preference here? I would say latest version is better, since it has newer versions of packages, especially `sphinx` itself.

Reply:
I agree, for documentation let's stick with what it works!